Kudu.Services/Diagnostics/ProcessController.cs

```diff
@@ -456,6 +456,12 @@ private ProcessInfo GetProcessInfo(Process process, string href, bool details =
                 SetEnvironmentInfo(info);
             }
 
+            if (ArmUtils.IsArmRequest(Request))
```

Comment:
Why not just `if (!ArmUtils.IsArmRequest(Request))` in the code above where you set them in the first place?

Reply:
I just mentioned that here https://github.com/projectkudu/kudu/pull/2868#discussion_r219310329 
 
> we could have populating these as a conditional above, but then we will lose the info for
```
processInfo.IsScmSite 
processInfo.IsWebJob 
processInfo.Description
```
>which are deduced from the environment variables.